gn3/db/rdf/wiki.py

```diff
@@ -180,3 +187,112 @@ def get_comment_history(
+
+    INSERT {
+            GRAPH <http://genenetwork.org> {
+            [ rdfs:label '''XXXX'''@en] rdf:type gnc:GNWikiEntry ;
+                    gnt:symbol "XXXX" ;
+                    foaf:mbox <XXXX> ;
+                    gnt:initial "XXXX" ;
+                    gnt:belongsToSpecies ?speciesId ;
+                    gnt:reason "XXXX" ;
+                    foaf:homepage <XXXX> ;
+                    dct:references pmid:XXXX ;
+                    dct:references pmid:XXXX ;
+                    gnt:belongsToCategory "XXXX";
+                    gnt:belongsToCategory "XXXX";
+                    dct:hasVersion "123"^^xsd:integer ;
+                    dct:identifier "1"^^xsd:integer ;
+                    dct:created "2024-09-11 11:00"^^xsd:datetime .
+
+            } USING <http://genenetwork.org> WHERE {
+                    ?speciesId gnt:shortName "{species}" .
+
+            }
+    }
+    """
+    next_version = get_next_comment_version(
+        comment_id, sparql_conf['sparql_uri'], graph)
+    name = f"gn:wiki-{comment_id}-{next_version}"
+    comment_triple = Template("""$name rdf:label '''$comment'''@en ;
+rdf:type gnc:GNWikiEntry ;
+gnt:symbol "$symbol" ;
+dct:identifier "$comment_id"^^xsd:integer ;
+dct:hasVersion "$next_version"^^xsd:integer ;
+dct:created "$created"^^xsd:datetime .
+""").substitute(
+        comment=payload["comment"],
+        name=name, symbol=payload['symbol'],
+        comment_id=comment_id, next_version=next_version,
+        created=datetime.datetime.now(datetime.timezone.utc).strftime('%Y-%m-%d %H:%M:%S'))
+    using = ""
+    if payload["email"]:
+        comment_triple += f"{name} foaf:mbox <{payload['email']}> .\n"
+    if payload["initial"]:
+        comment_triple += f"{name} gnt:initial \"{payload['initial']}\" .\n"
+    if payload["species"]:
+        comment_triple += f"{name} gnt:belongsToSpecies ?speciesId .\n"
+        using = Template(
+            """ USING $graph WHERE { ?speciesId gnt:shortName "$species" . } """).substitute(
+                graph=graph, species=payload["species"]
+        )
+    if payload["reason"]:
+        comment_triple += f"{name} gnt:reason \"{payload['reason']}\" .\n"
+    if payload["web_url"]:
+        comment_triple += f"{name} foaf:homepage <{payload['web_url']}> .\n"
+    for pmid in payload["pubmed_ids"]:
+        comment_triple += f"{name} dct:references pubmed:{pmid} .\n"
+    for category in payload["categories"]:
+        comment_triple += f'{name} gnt:belongsToCategory "{category}".\n'
+
+    res = update_rdf(
+        query=Template(
+            """
+$prefix
+
+INSERT {
+GRAPH $graph {
+$comment_triple}
+} $using
+""").substitute(prefix=RDF_PREFIXES,
+                graph=graph,
+                comment_triple=comment_triple,
+                using=using),
+        sparql_user=sparql_conf["sparql_user"],
+        sparql_password=sparql_conf["sparql_password"],
+        sparql_auth_uri=sparql_conf["sparql_auth_uri"],
+    )
+    return (res, 200)
```

Comment:
why are we returning 200 here?

Reply:
No good reason.  I reckon we can just return the `res` only.